eng/common/TestResources/New-TestResources.ps1

```diff
@@ -294,10 +294,11 @@ function BuildDeploymentOutputs([string]$serviceName, [object]$azContext, [objec
         "AZURE_SERVICE_DIRECTORY" = $serviceName.ToUpperInvariant();
     }
 
+    $deploymentOutputs["${serviceDirectoryPrefix}TENANT_ID"] = $azContext.Tenant.Id;
```

Comment:
I'd like to see this go behind a flag, if we're intending to keep it.   This needs to be able to run clean for folks after a fresh clone of the repo - and outside of the Microsoft tenant.

Let's please test in a non-MS environment.

Reply:
I only made this change to test that setting these variables would fix some issues. I've since reverted the change to New-TestResources.ps1 and moved setting the variables in live.tests.yml as @weshaggard suggested to ensure they are set when running live tests in the pipeline.